Sources/Player.swift

```diff
@@ -441,7 +458,7 @@ extension Player {
         let keys: [String] = [PlayerTracksKey, PlayerPlayableKey, PlayerDurationKey]
 
         self._asset.loadValuesAsynchronously(forKeys: keys, completionHandler: { () -> Void in
-            DispatchQueue.main.sync(execute: { () -> Void in
+            DispatchQueue.main.async(execute: { () -> Void in
```

Comment:
hey @eliotfowler thanks for the PR. I haven't had a chance to look through it but will when i get the chance.

The sync here is ensuring the the code is executed on the main thread. Since it's being called within the outer async func, it's already async. Making the inner async again just executes that closure later and out of order for the keys. I'd prefer that we not change this.